src/test/resources/catalogs/default/EXCLUDE_T_RESERVED_KEYWORDS.ion

```diff
@@ -0,0 +1,56 @@
+{
+  type: "bag",
+  items: {
+    type: "struct",
+    constraints: [ closed, ordered, unique ],
+    fields: [
+      {
+        name: "flds",
+        type: {
+          type: "struct",
+          constraints: [ closed, ordered, unique ],
+          fields: [
+            {
+              name: "select",
+              type: {
+                type: "struct",
+                constraints: [ closed, ordered, unique ],
+                fields: [
+                  {
+                    name: "field_x",
+                    type: "int32",
+                  },
+                  {
+                    name: "field_y",
+                    type: "string",
+                  }
+                ]
+              },
+            },
+            {
+              name: "order",
```

Comment:
(self-review): `select` and `order` are both reserved keywords in Trino and PartiQL
- https://trino.io/docs/current/language/reserved.html
- https://partiql.org/syntax/reserved-keywords-spec-version-2019.html#_reserved_keywords